certified-connectors/Africa's Talking SMS/apiDefinition.swagger.json

```diff
@@ -210,6 +210,265 @@
         },
         "x-ms-visibility": "important"
       }
+    },
+    "/api/v1/powerapps/{username}/sms/dlr/callback": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "id"
+            },
+            "status": {
+              "type": "string",
+              "description": "status"
+            },
+            "phoneNumber": {
+              "type": "string",
+              "description": "phoneNumber"
+            },
+            "networkCode": {
+              "type": "string",
+              "description": "networkCode"
+            },
+            "failureReason": {
+              "type": "string",
+              "description": "failureReason"
+            },
+            "retryCount": {
+              "type": "integer",
+              "format": "int32",
+              "description": "retryCount"
+            }
+          }
+        },
+        "headers": {
+          "Content-type": {
+            "description": "Content-type",
+            "type": "string"
+          }
+        },
+        "description": "A new delivery report for a message"
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200"
```

Comment:
@crispinkoech Can you please provide a valid description for the "200" response code instead of "200"